gas/config/tc-riscv.c

```diff
@@ -171,6 +182,13 @@ riscv_set_arch (const char *s)
       case 'i':
 	break;
 
+      case 'e':
+	p++;
+	riscv_add_subset ("e");
+	riscv_add_subset ("i");
+	riscv_set_rve (TRUE);
```

Comment:
I think we want to add

    if (xlen > 32)
      as_fatal ("-march=%s: rv%de is not a valid base ISA", s, xlen);